.travis.yml

```diff
@@ -2,44 +2,36 @@ dist: xenial
 sudo: required
 
 language: go
-go_import_path: github.com/opensds/multi-cloud
+go_import_path: github.com/sodafoundation/multi-cloud
+
 go:
   - 1.13
   - tip
 
-services:
-  - docker
-  - docker-compose
-
 env:
-  matrix:
   - TARGET=amd64
     GO111MODULE=on
-  global:
-  - secure: "IhEskTMEAsBNYTQCpDSlvDxIJK5cu1r2V"
+
+services:
+  - docker
 
 before_install:
-  - wget -qO - https://packages.confluent.io/deb/5.3/archive.key | sudo apt-key add -
-  - wget -q -O- "https://download.ceph.com/keys/release.asc" | sudo apt-key add -
-  - sudo add-apt-repository "deb [arch=amd64] https://packages.confluent.io/deb/5.3 stable main"
```

Comment:
Why to remove kafka installation? multi-cloud need it

Reply:
not needed right now, will add when needed.

Reply:
Kafka is required for multi-cloud. So suggest, please keep it